routes/v1.js

```diff
@@ -33,12 +34,32 @@ apiRoutes.route('/verify/:verificationId')
   .get(UserController.verifyAccount);
 
 apiRoutes.route('/articles')
-  .post(Auth.verifyUser, isUserVerified,
+  .post(
+    Auth.verifyUser, isUserVerified,
     addImages,
     validateArticle,
     returnValidationErrors,
     generateSlug,
-    createArticle);
+    createArticle
+  );
+
+apiRoutes.route('/articles/:id')
+  .put(
```

Comment:
Please consider using a `patch` method here. Put supposes you are sending the entire resource body. See more here: https://stackoverflow.com/questions/28459418/rest-api-put-vs-patch-with-real-life-examples

Reply:
Done